src/dawn/CodeGen/GridTools/GTCodeGen.cpp

```diff
@@ -387,6 +387,9 @@ GTCodeGen::generateStencilInstantiation(const StencilInstantiation* stencilInsta
 
     std::vector<std::string> makeComputation;
 
+    // map field names to maximum extent 
+    std::map<std::string, std::vector<int>> collectedExtents;
```

Comment:
this seems to be a leftover, remove?

Reply:
fixed with https://github.com/MeteoSwiss-APN/dawn/pull/96/commits/6e09d5205ce94885f57ba4c9620488826e682232